### ISSUE TITLE
Run an updated build to include a recent bugfix

### ISSUE THREAD

**Issue by heatherhosler** [Plattar/python-usd-ar#7]
I am having a problem with [this bug](https://github.com/PixarAnimationStudios/OpenUSD/issues/1902).
It has been fixed in a recent version of OpenUSD.
However, this repo's most recent build was last year.

Any chance of triggering another build using OpenUSD v23.11, which is the latest?

Many thanks, while waiting, I will clone & try to figure out how to get it working, although I've not used GitHub workflows before so I don't exactly know what I'm doing.

**Comment by heatherhosler** [comment]
I found the upstream repo `python-usd` where I submitted a [matching issue](https://github.com/Plattar/python-usd/issues/9).

**Comment by DavidArayan** [comment]
Yes, we need to update this for our internal usage aswell. Should be a fairly straightforward update, i'll add it to our backlog.

**Comment by DavidArayan** [comment]
[matching issue (python-usd issue 9)](https://github.com/Plattar/python-usd/issues/9) has been merged and a build using OpenUSD 23.11 is now available. I will re-build this repository using the new version, the build will take several hours so a new version should be up within 12-24 hours. I'll link a PR once its ready.

**Comment by sreyemnayr** [comment]
I see that an x86/64 version got pushed to dockerhub but not an arm version... Also no update to this repo (I guess the only real change is the base image?) Wonder if this is forthcoming. I'm not having success building the image myself on my M2. Thanks!

**Comment by DavidArayan** [comment]
We're having the same issue with building for ARM images, investigating as to why.
